pkg/clients/openshift.go

```diff
@@ -41,13 +42,20 @@ import (
 	rbac "k8s.io/kubernetes/pkg/apis/rbac"
 )
 
+const (
+	// ChangePodNetworkAnnotation - Annotation used for changing a pod
+	// network, used to join networks together.
+	ChangePodNetworkAnnotation = "pod.network.openshift.io/multitenant.change-network"
```

Comment:
ah, right... I was going to say "that constant should be available in `networkoapi`", but it's not, because we had decided that this is "internal API", not public API. So maybe that's an argument that you should use `oc adm pod-network` instead? (But the API is still guaranteed to keep existing, since we have to preserve compatibility between old/new clients/servers, so it's safe for you to depend on it if you'd rather keep the code this way.)

Reply:
I think that we are attempting to remove shelling out in our code, therefore I think on a whole we would prefer to keep using the API endpoints. 

@jmrodri @eriknelson can you guys weigh in as far as broker code base is concerned? It sounds like the question is if we have to duplicate some constants and some logic around checking if an annotation has been updated vs shelling out to the oc command.

@danwinship Is that a fair characterization of the concern here? Do you see more code duplication then the above mentioned?